src/Config/ObjectModel/RuntimeOptions.cs

```diff
@@ -63,6 +63,12 @@ GraphQL is null ||
         GraphQL?.Enabled is null ||
         GraphQL?.Enabled is true;
 
+    [JsonIgnore]
+    public bool IsMcpEnabled =>
```

Comment:
If we have `IsMcpEnabled` in the `RuntimeConfig` class why do we need it again in the `RuntimeOptions` class?